packages/desktop-client/src/components/manager/subscribe/Login.tsx

```diff
@@ -1,13 +1,16 @@
 // @ts-strict-ignore
-import React, { type ChangeEvent, useState } from 'react';
+import { useState, useEffect } from 'react';
```

Comment:
```suggestion
import React, { type ChangeEvent, useState, useEffect } from 'react';
```

Missing some changes from the default branch